api/Document.json

```diff
@@ -8765,7 +8785,7 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true,
+              "version_added": false,
```

Comment:
Can you revert this change? (This is `document.popupNode`.) The note makes it clear this isn't web exposed. I don't think we should have this in BCD, but it's a different kind of change to the rest in this PR.